src/main/java/seedu/modtrek/logic/commands/TagCommand.java

```diff
@@ -46,7 +46,8 @@ public class TagCommand extends Command {
      */
     public static final String MESSAGE_REMOVE_TAG_SUCCESS = "Removed tag from Module: %1$s";
 
-    public static final String MESSAGE_MISSING_PREFIX = "Missing tag prefix /t.";
+    public static final String MESSAGE_MISSING_PREFIX = "Missing tag prefix /t.\nExample: tag cs2109s include /t CSF "
```

Comment:
I think if wanna add an example here, can add another newline "\n" before the example to standardise with message usage and "MESSAGE_INVALID_COMMAND_FORMAT".